src/node/utils.ts

```diff
@@ -7,3 +7,24 @@ export function buildLog(text: string, count?: number) {
 export function getSize(str: string) {
   return `${(str.length / 1024).toFixed(2)}kb`
 }
+
+export function collectRoutePaths(routes: any[]) {
```

Comment:
```suggestion
export function routesToPaths(routes: any[]) {
```

WDYT?

Reply:
Ah, and we should add the type for routes.

Reply:
Yes 👍